src/Xamarin.Android.Tools.Bytecode/Tests/ClassFileFixture.cs

```diff
@@ -25,18 +25,22 @@ protected static string LoadString (string resource)
 				return r.ReadToEnd ();
 		}
 
-		protected static void AssertXmlDeclaration (string classResource, string xmlResource, string documentationPath = null)
+		protected static void AssertXmlDeclaration (string classResource, string xmlResource, string documentationPath = null, JavaDocletType? javaDocletType = null)
 		{
 			var classPathBuilder    = new ClassPath () {
 				ApiSource           = "class-parse",
 				DocumentationPaths  = new string[] {
 					documentationPath,
 				},
 			};
+			if (javaDocletType.HasValue)
+				classPathBuilder.DocletType = javaDocletType.Value;
 			classPathBuilder.Add (LoadClassFile (classResource));
 
 			var actual  = new StringWriter ();
 			classPathBuilder.ApiSource  = "class-parse";
+			if (javaDocletType.HasValue)
```

Comment:
I guess they are just extraneous, but it was @dellis1972 who wrote this class-parse bits so he might have some comments.